KeepingYouAwake/fr.lproj/Localizable.strings

```diff
@@ -32,4 +32,3 @@ Click droit pour le menu";
 
 /* Version */
 "Version" = "Version";
-
```

Comment:
I personally prefer having a empty line at the end of source and `.strings` files.